DailyReportWeb/Controllers/Api/AccountController.cs

```diff
@@ -20,6 +20,7 @@
 namespace DailyReportWeb.Controllers.Api
 {
     [Authorize]
+    [ExceptionHandling]
```

Comment:
Let's put this on all the controllers. So that we cover all exceptions. Maybe create a BaseApiController that will be inherited in all the other api controllers, and put the attribute on that.
Also put this on the mvc controllers also.